File-Formats/Presentation/Presentation-to-PDF.md

```diff
@@ -1258,6 +1258,57 @@ pdfDoc.Close()
 
 {% endtabs %}
 
+## Recreate Nested Metafile
+
+This setting allows you to regenerate the nested EMF images present in the PowerPoint presentation document during PDF conversion.
+This property is recommended to resolve the scaling problem of nested metafile images by regenerating the nested metafile images present in the PowerPoint presentation document.
+
+The following code sample shows how to use this property to regenerate the nested EMF images present in the PowerPoint presentation document during PDF conversion.
```

Comment:
The following code sample shows how to use this property to regenerate the nested EMF images in the PowerPoint presentation document during PDF conversion.

Reply:
Feedback addressed